libs/core/src/lib/shellbar/shellbar-actions/shellbar-actions.component.scss

```diff
@@ -0,0 +1,13 @@
+.fd-shellbar-actions-combobox--mobile {
```

Comment:
Would be nice to have in the css repo

Reply:
https://github.com/SAP/fundamental-styles/pull/4012
@mikerodonnell89